Assets/Scripts/Framework/Gameplay/Interactions/Grabbables/Grabbable.cs

```diff
@@ -20,20 +21,31 @@ public abstract class Grabbable : Interactable
         private Interactor _originInteractor;
 
         /// <summary>
-        /// Returns the <see cref="Interactor"/> of this Interactable, will be null if not being 
+        /// Returns the <see cref="Interactor"/> of this Interactable, will be null if not being
         /// targeted/interacted with.
         /// </summary>
         public Interactor OriginInteractor => _originInteractor;
 
         [field: SyncVar(Channel = FishNet.Transporting.Channel.Reliable)]
         private protected bool IsBeingInteracted { get; set; }
 
+        /// <summary>
+        /// <inheritdoc/>
+        /// </summary>
+        /// <param name="connection"><inheritdoc/></param>
+        public override void OnSpawnServer(NetworkConnection connection)
+        {
+            base.OnSpawnServer(connection);
+
+            PlaceOnGround();
+        }
```

Comment:
Does this work? When I tested this it didn't work so I went back to private void Start().